.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+branches:
+    only:
+        - master
+
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - '5.5'
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - nightly
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: DEPS=dev
```

Comment:
ah, so you mean this can be used instead of having `~5.3.0@dev` for e.g.? And then would apply to every package right?